rules/misc_rules.build_defs

```diff
@@ -283,11 +283,19 @@ def hash_filegroup(name:str, srcs:list=None, deps:list=None, exported_deps:list=
       test_only (bool): If true the exported file can only be used by test targets.
       requires (list): Kinds of output from other rules that this one requires.
     """
+    cmd = 'mkdir _out && '
+    cmd += 'SHA256CMD="$(command -v sha256sum || echo shasum)" && '
```

Comment:
sha256sum is part of coreutils right? If so, I don't think we need to have this switch.